### ISSUE TITLE
GraphServiceClient.chats.post(body): 400 bad request (chat members cannot be null)

### ISSUE THREAD

**Issue by krusli** [microsoftgraph/msgraph-sdk-python#319]
I'm trying to create a new one-on-one chat using the Graph API. This works just fine on the Graph Explorer, but when I try to do it using the library I get this error:

```ODataError: APIError 400: None MainError(additional_data={}, code='BadRequest', details=None, inner_error=InnerError(additional_data={'date': datetime.timedelta(seconds=46011)}, client_request_id='', date=None, odata_type=None, request_id=''), message="Chat members cannot be null. (Parameter 'Members')", target=None)```

I'm not sure how to debug/print the outgoing HTTP request yet, as I'm new to using the Python library, but I tried making my own writer, calling `serialize()` with the writer on the `Chat` object, and the members are indeed populated there.

On another note, the sample in Graph Explorer and the docs seems to be outdated (there is no argument called `request_body`, only `body`), and since the imports are not in any of the code samples, it was really annoying to have to hunt down the correct imports using Python's `dir` and `inspect.signature`.

<img width="1046" alt="image" src="https://github.com/microsoftgraph/msgraph-sdk-python/assets/14309096/6a185569-bfed-40cf-a0fe-1a49a7b63088">

---

Edit: Managed to print the request content by setting up custom middleware (https://learn.microsoft.com/en-us/graph/sdks/customize-client?tabs=csharp). It seems to be an empty `{}`. I get the same result on Graph Explorer if the request body is `{}`

---

Edit 2: the same thing is happening on `result = await graph_client.chats.by_chat_id('chat-id').messages.post(body = request_body)`, the body that gets passed is blank. I am using `1.0.0a14` for reference.

**Comment by krusli** [comment]
Downgrading to `1.0.0a13` seems to fix the issue.

**Comment by krusli** [comment]
<img width="630" alt="image" src="https://github.com/microsoftgraph/msgraph-sdk-python/assets/14309096/5afbba66-b1cb-4fac-b856-9a9884e6ccd7">

The auto-generated snippet also misses the comma before `additional_data`, and it should be `user@odata.bind` on the key, not `user@odata.bind`.

**Comment by samwelkanda** [comment]
Hi @krusli 

Thanks for reporting this issue. This was a bug affecting POST/PUT/PATCH requests due to the way we track changes to models in our abstractions library.
The issue was resolved in the latest version `1.0.0a15` and the underlying dependency  `microsoft-kiota-abstractions==0.8.5`

I have also confirmed the issues on the snippets and have raised them with the docs team. 

Thank you for the valuable feedback.
